packages/selected-network-controller/CHANGELOG.md

```diff
@@ -7,6 +7,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [9.0.0]
+
+### Changed
+
+- **BREAKING** remove `perDomainNetwork` from state ([#3989](https://github.com/MetaMask/controllers/pull/3989))
```

Comment:
Following the merge of #4000 it would be good to mention the addition of the dependency / peer dependency here:

``` suggestion
- **BREAKING** remove `perDomainNetwork` from state ([#3989](https://github.com/MetaMask/controllers/pull/3989))
- **BREAKING** Add dependency and peer dependency on `@metamask/permission-controller` ^8.0.1 ([#4000](https://github.com/MetaMask/controllers/pull/4000))